src/index.js

```diff
@@ -56,6 +56,7 @@ export default (elementId, options = {}) => {
 
     playerAPI = YouTubePlayer.promisifyPlayer(playerAPIReady);
     playerAPI.on = emitter.on;
+    playerAPI.off = emitter.off;
```

Comment:
I don't know what I would have to document, I only added a single line. It is very similar to the assignment right above it